### ISSUE TITLE
Unable to quickly open note with large pasted-in image

### ISSUE THREAD

**Issue by SidharthBansal** [publiclab/plots2#2382]
### Please describe the problem (or idea)
The pages like https://publiclab.org/notes/bansal_sidharth2996/02-20-2018/gsoc-proposal-oauth-authentication?_=1519141316  take more time to load than other pages containing no or small sized images.
See #2370 for the previous discussion

**Comment by SidharthBansal** [comment]
One solution could be not allowing users to post in large images as inline data. Having a validation check on the size of images say 5MB?

**Comment by jywarren** [comment]
Yes, i think that's not a bad idea. Should we make this an option in
https://github.com/publiclab/PublicLab.Editor/ that can be enabled to limit
it? Maybe validation on by default? Then we can update it in Bower from
plots2.

How does that sound?

**Comment by SidharthBansal** [comment]
I have downloaded the images which I pasted there. When I checked their size one was of 64.9kb and other of 127 kb. So, I think the opening up of the research page not due to the image's size. What do you say?
